distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -943,6 +944,11 @@ type Date
         options = [Option "<Fixed Date>" "Date.new"] + (if include_today then [Option "<Today>" "Date.today"] else [])
         Widget.Single_Choice values=options display=display
 
+    > self that:Date = Comparable.> self that
```

Comment:
- The idea here, @jdunkerley, is to define these four methods in every type that is _truly comparable_
- rather than providing own implementation, only delegate to `Comparable`'s default implementation
- e.g. the _comparing functionality_ shall remain the same as before
- it is just not exposed on every object